src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/UseContainerLevelAccessPolicyTests.cs

```diff
@@ -66,6 +66,36 @@ public void TestMethod(SharedAccessBlobPolicy policy, SharedAccessBlobHeaders he
             GetCSharpResultAt(12, 9));
         }
 
+        [Fact]
+        public async Task TestPropertyInitializerGroupPolicyIdentifierOfBlobNamespaceIsNullDiagnostic()
```

Comment:
Interestingly, the tests in this file passed in the 2.9.x branch as-is. I guess PointsToAnalysis already worked for initializers?

Reply:
Interesting - can you check if the Operation tree in your test has an implicit IBlockOperation somewhere?

Reply:
Ah actually I look at the analyzer and it seems lack of CFG makes it more aggressive - you probably need to write reverse tests where you are not expecting a diagnostic?

Reply:
Oh, hrm, maybe I don't like that analyzer's behavior when it can't find a CFG.